File: Data Structures & Algorithms/duplicate-integer/submission-2.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        if (nums.length == 1) {
            return false;
        }
        HashSet<Integer> hs = new HashSet<>();
        for (int i: nums) {
            if (hs.contains(i)) {
                return true;
            }
            hs.add(i);
        }
        return false;
    }
}

File: Data Structures & Algorithms/is-anagram/submission-2.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        a, b = {}, {}
        for i in s:
            a[i] = a.get(i, 0) + 1
        for j in t:
            b[j] = b.get(j, 0) + 1
        if a == b:
            return True
        return False